pkgs/tools/text/kenlm/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, cmake, boost, eigen, xz, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  pname = "kenlm";
+  version = "unstable-2020-11-03";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "kpu";
+    repo = "${pname}";
```

String interpolation is not required here.
```suggestion
    repo = pname;
```
(IMO just using  `repo = "kenlm";` looks cleaner, but that's just preference)